backend/chalice/api_server/.chalice/config.json.dev

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
isn't this file generated using https://github.com/chanzuckerberg/corpora-data-portal/blob/7e9afe90d9589b850d0dabcf225e239af30f92e7/backend/chalice/api_server/Makefile#L49. How is this different?

Reply:
The build-chalice-config target has a dependency on being able to talk to AWS:
https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/chalice/api_server/Makefile#L13

The dockerized local dev environment is designed to have no (or as few as possible) external dependencies, so we can't depend on making AWS calls to populate it. This template just gives us a simple way to sidestep that dependency, until we get more comfortable with the docker dev env.